src/m365/spo/commands/contenttype/contenttype-get.ts

```diff
@@ -43,7 +54,12 @@ class SpoContentTypeGetCommand extends SpoCommand {
       .get(requestOptions)
       .then((res: any): void => {
         if (res['odata.null'] === true) {
-          cb(new CommandError(`Content type with ID ${args.options.id} not found`));
+          if(args.options.id){
+            cb(new CommandError(`Content type with ID ${args.options.id} not found`));
+          }
+          if(args.options.contenttypeTitle){
+            cb(new CommandError(`Content type with title ${args.options.contenttypeTitle} not found`));
+          }
           return;
         }
 
```

Comment:
Depending if you look up content type by its name or ID, you get two different response types. When you look up content type by ID, you get back the content type object. On the other hand, when you look it up by name, you get back `{ value: [ contentTypeObjectHere ] }`. We should not pass the raw response to the command output because it would make the same command have different output depending which option was used. Instead, when the user chose to retrieve a content type by name, we should extract the retrieved content type object from the `value` array and pass it to output so that the command's output is consistent in both cases.

Reply:
ok corrected, now when we do get content type by id or name we see the same response type
```
PS C:\workspace\cli-microsoft365\cli-microsoft365> m365 spo contenttype get --webUrl https://tenanttocheck.sharepoint.com/sites/ModernSite --name "CustomContentType"
Description              : test content type
DisplayFormTemplateName  : ListForm
DisplayFormUrl           :
DocumentTemplate         :
DocumentTemplateUrl      :
EditFormTemplateName     : ListForm
EditFormUrl              :
Group                    : CustomContentType
Hidden                   : false
Id                       : {"StringValue":"0x0100F7C90FF10BC14E408CD9FF01890596D2"}
JSLink                   :
MobileDisplayFormUrl     :
MobileEditFormUrl        :
MobileNewFormUrl         :
Name                     : CustomContentType
NewFormTemplateName      : ListForm
NewFormUrl               :
ReadOnly                 : true
SchemaXml                : <ContentType ID="0x0100F7C90FF10BC14E408CD9FF01890596D2" Name="CustomContentType" Group="CustomContentType" Description="test content type" ReadOnly="TRUE" Version="0"><Folder TargetName="_cts/CustomContentType" /><Fields><Field ID="{c042a256-787d-4a6f-8a8a-cf6ab767f12d}" Name="ContentType" SourceID="http://schemas.microsoft.com/sharepoint/v3" StaticName="ContentType" Group="_Hidden" Type="Computed" DisplayName="Content Type" Sealed="TRUE" Sortable="FALSE" RenderXMLUsingPattern="TRUE" PITarget="MicrosoftWindowsSharePointServices" PIAttribute="ContentTypeID" DelayActivateTemplateBinding="GROUP,SPSPERS,SITEPAGEPUBLISHING" Customization=""><FieldRefs><FieldRef ID="{03e45e84-1992-4d42-9116-26f756012634}" Name="ContentTypeId" /></FieldRefs><DisplayPattern><MapToContentType><Column Name="ContentTypeId" /></MapToContentType></DisplayPattern></Field><Field ID="{fa564e0f-0c70-4ab9-b863-0177e6ddd247}" Name="Title" SourceID="http://schemas.microsoft.com/sharepoint/v3" StaticName="Title" Group="_Hidden" Type="Text" DisplayName="Title" Required="TRUE" FromBaseType="TRUE" DelayActivateTemplateBinding="GROUP,SPSPERS,SITEPAGEPUBLISHING" Customization="" ShowInNewForm="TRUE" ShowInEditForm="TRUE" /><Field DisplayName="TestColumn" Type="Boolean" Required="FALSE" ID="{15d33cdd-06bc-4a10-86d7-85261f0bad5c}" SourceID="{f57b4a1a-e38e-4108-b85a-93d4a811ba6f}" StaticName="TestColumn" Name="TestColumn" Group="CustomColumns" CustomFormatter="" Description="Custom column in custom content type" EnforceUniqueValues="FALSE" Customization=""><Default>0</Default></Field></Fields><XmlDocuments><XmlDocument NamespaceURI="Microsoft.SharePoint.Taxonomy.ContentTypeSync"><SharedContentType xmlns="Microsoft.SharePoint.Taxonomy.ContentTypeSync" SourceId="ee50f6a5-f801-4f27-8383-acf4b8b089d8" ContentTypeId="0x0100F7C90FF10BC14E408CD9FF01890596D2" PreviousValue="false" /></XmlDocument><XmlDocument NamespaceURI="http://schemas.microsoft.com/sharepoint/v3/contenttype/forms"><FormTemplates xmlns="http://schemas.microsoft.com/sharepoint/v3/contenttype/forms"><Display>ListForm</Display><Edit>ListForm</Edit><New>ListForm</New></FormTemplates></XmlDocument></XmlDocuments></ContentType>
Scope                    : /sites/ModernSite
Sealed                   : false
StringId                 : 0x0100F7C90FF10BC14E408CD9FF01890596D2



PS C:\workspace\cli-microsoft365\cli-microsoft365> m365 spo contenttype get --webUrl https://tenanttocheck.sharepoint.com/sites/ModernSite --id 0x0100F7C90FF10BC14E408CD9FF01890596D2
Description              : test content type
DisplayFormTemplateName  : ListForm
DocumentTemplate         :
DocumentTemplateUrl      :
EditFormTemplateName     : ListForm
EditFormUrl              :
Group                    : CustomContentType
Hidden                   : false
Id                       : {"StringValue":"0x0100F7C90FF10BC14E408CD9FF01890596D2"}
JSLink                   :
MobileDisplayFormUrl     :
MobileEditFormUrl        :
MobileNewFormUrl         :
Name                     : CustomContentType
NewFormTemplateName      : ListForm
NewFormUrl               :
ReadOnly                 : true
SchemaXml                : <ContentType ID="0x0100F7C90FF10BC14E408CD9FF01890596D2" Name="CustomContentType" Group="CustomContentType" Description="test content type" ReadOnly="TRUE" Version="0"><Folder TargetName="_cts/CustomContentType" /><Fields><Field ID="{c042a256-787d-4a6f-8a8a-cf6ab767f12d}" Name="ContentType" SourceID="http://schemas.microsoft.com/sharepoint/v3" StaticName="ContentType" Group="_Hidden" Type="Computed" DisplayName="Content Type" Sealed="TRUE" Sortable="FALSE" RenderXMLUsingPattern="TRUE" PITarget="MicrosoftWindowsSharePointServices" PIAttribute="ContentTypeID" DelayActivateTemplateBinding="GROUP,SPSPERS,SITEPAGEPUBLISHING" Customization=""><FieldRefs><FieldRef ID="{03e45e84-1992-4d42-9116-26f756012634}" Name="ContentTypeId" /></FieldRefs><DisplayPattern><MapToContentType><Column Name="ContentTypeId" /></MapToContentType></DisplayPattern></Field><Field ID="{fa564e0f-0c70-4ab9-b863-0177e6ddd247}" Name="Title" SourceID="http://schemas.microsoft.com/sharepoint/v3" StaticName="Title" Group="_Hidden" Type="Text" DisplayName="Title" Required="TRUE" FromBaseType="TRUE" DelayActivateTemplateBinding="GROUP,SPSPERS,SITEPAGEPUBLISHING" Customization="" ShowInNewForm="TRUE" ShowInEditForm="TRUE" /><Field DisplayName="TestColumn" Type="Boolean" Required="FALSE" ID="{15d33cdd-06bc-4a10-86d7-85261f0bad5c}" SourceID="{f57b4a1a-e38e-4108-b85a-93d4a811ba6f}" StaticName="TestColumn" Name="TestColumn" Group="CustomColumns" CustomFormatter="" Description="Custom column in custom content type" EnforceUniqueValues="FALSE" Customization=""><Default>0</Default></Field></Fields><XmlDocuments><XmlDocument NamespaceURI="Microsoft.SharePoint.Taxonomy.ContentTypeSync"><SharedContentType xmlns="Microsoft.SharePoint.Taxonomy.ContentTypeSync" SourceId="ee50f6a5-f801-4f27-8383-acf4b8b089d8" ContentTypeId="0x0100F7C90FF10BC14E408CD9FF01890596D2" PreviousValue="false" /></XmlDocument><XmlDocument NamespaceURI="http://schemas.microsoft.com/sharepoint/v3/contenttype/forms"><FormTemplates xmlns="http://schemas.microsoft.com/sharepoint/v3/contenttype/forms"><Display>ListForm</Display><Edit>ListForm</Edit><New>ListForm</New></FormTemplates></XmlDocument></XmlDocuments></ContentType>
Scope                    : /sites/ModernSite
Sealed                   : false
StringId                 : 0x0100F7C90FF10BC14E408CD9FF01890596D2
```